Formula/alicevision.rb

```diff
@@ -91,12 +96,13 @@ def caveats; <<~EOS
  set(CMAKE_CXX_STANDARD 11)
  set(CMAKE_CXX_STANDARD_REQUIRED ON)
  
-@@ -794,6 +794,7 @@ set(ALICEVISION_INCLUDE_DIRS
+@@ -794,6 +794,8 @@ set(ALICEVISION_INCLUDE_DIRS
          ${CMAKE_CURRENT_SOURCE_DIR}
          ${generatedDir}
          ${CMAKE_CURRENT_SOURCE_DIR}/dependencies
 +        ${CMAKE_CURRENT_SOURCE_DIR}/dependencies/MeshSDFilter
          ${LEMON_INCLUDE_DIRS}
++        /usr/local/Cellar/eigen/3.3.9/include/eigen3/
```

Comment:
This is very dirty; I'm not sure why this broke, but it's probably not worth looking into it unless it remains an issue in #12.